src/react/databrowser/objects/Objects.tsx

```diff
@@ -127,6 +132,10 @@ export default function Objects() {
   // }
   return (
     <L.ContentContainer>
```

Comment:
While I am seeing the changes in the ListLayoutx.ts, I am wondering if we could remove them as much as possible given we have out new Layout Component. 
For example, if we could get rid of this `<L.ContentContainer/>`,